_config.yml

```diff
@@ -38,4 +38,9 @@ sidebar:
   - name: PySigma Converter
     icon: <i class="fa-solid fa-screwdriver-wrench"></i>
     link: https://github.com/SigmaHQ/pySigma
- 
\ No newline at end of file
+ 
+# https://github.com/github/pages-gem/issues/399#issuecomment-301827749
+# When running locally, we run into the following error —
+# GitHub Metadata: No GitHub API authentication could be found. Some fields may be missing or have incorrect data.
+# adding this avoids it
```

Comment:
```suggestion
# https://github.com/github/pages-gem/issues/399#issuecomment-301827749
# When running locally, we run into the following error —
# GitHub Metadata: No GitHub API authentication could be found. Some fields may be missing or have incorrect data.
# Adding the following line to avoid the issue
```